src/components/pages/FriendItem/FriendItem.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import friendShape from '../../../helpers/propz/friendShape';
+import './FriendItem.scss';
+
+class FriendItem extends React.Component {
+  static propTypes = {
+    listing: friendShape,
```

Comment:
listing??? awkward copy paste

Reply:
LOL, promise it wasn't copied with my mouse, just my fingers. I'll change that. :)

Reply:
Mkay I fixed my mistake in the latest commit!